thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/task/TaskConstants.java

```diff
@@ -0,0 +1,56 @@
+package com.linkedin.thirdeye.anomaly.task;
+
+public class TaskConstants {
+
+  public enum TaskType {
+    AnomalyDetection {
+
+      @Override
+      Class<?> getKlazz() {
+        return AnomalyDetectionTaskRunner.class;
```

Comment:
Providing executor object for a task type should not be responsibility of TaskType. There should be a JobExecutorFactory which should proved JobExecutor based on TaskType.